modules/registrars/realtimeregister/src/Assets/Js/renew.js

```diff
@@ -0,0 +1,13 @@
+const renewButton = document.querySelector('input[type="button"][data-toggle="modal"][data-target="#modalRenew"]');
+
+if (renewButton) {
+    renewButton.addEventListener('click', () => {
```

Comment:
kan dit niet ook met jquery?

Reply:
jwel, maar jquery is vies, lelijk, oud en bah